app/reducers/events.js

```diff
@@ -218,7 +234,7 @@ const mutate = joinReducers(mutateComments('events'), mutateEvent);
 export default createEntityReducer({
   key: 'events',
   types: {
-    fetch: [Event.FETCH, Event.FETCH_UPCOMING]
+    fetch: [Event.FETCH, Event.FETCH_PREVIOUS, Event.FETCH_UPCOMING]
```

Comment:
Is this setting up a listener for the events defined? At least that is what I assumed. If so, I believe we can remove Event.FETCH as it is never caught by the mutateEvent()-switch?